examples/03_Existing Models.ipynb

```diff
@@ -20,7 +46,565 @@
+    "    \n",
+    "    if x is not None:\n",
+    "        event_state = batt.event_state(x)\n",
+    "        if event_state[\"EOD\"] > 0.95:\n",
+    "            load = 1 # Discharge\n",
+    "        elif event_state[\"EOD\"] < 0.05:\n",
+    "            load = -1 # Charge\n",
+    "            \n",
+    "    return batt.InputContainer({'i': load})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now simulate to the threshold and print the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "simulated_results = batt.simulate_to_threshold(future_loading, **config)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We will now plot the inputs, outputs, and event states. In the input plot, we can see the current drawn fluctuate between -1 and 1 based on the current load we defined in the future loading function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = simulated_results.inputs.plot(xlabel='time (s)', ylabel='current drawn (amps)', title='BatteryElectroChem Input')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the output plots, we can see changes in the voltage and temperature."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "fig = simulated_results.outputs.plot(keys=['v'], xlabel='time (s)', ylabel='voltage (V)', figsize=(10,4), title='BatteryElectroChem Outputs')\n",
+    "fig2 = simulated_results.outputs.plot(keys=['t'], xlabel='time (s)', ylabel='temperature (°C)', figsize=(10,4))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the event states plot, we can see `EOD` incrementally spiking and `InsufficientCapacity` linearly declining until it reaches 0, or when the event has occurred."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "fig = simulated_results.event_states.plot(xlabel='time (s)', ylabel='event states', labels={'EOD', 'InsufficientCapacity'}, title='BatteryElectroChem Event States')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Simplified Battery"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "`SimplifiedBattery` is a model from [[Sierra 2019]](https://www.sciencedirect.com/science/article/abs/pii/S0951832018301406). It was initially introduced in the __[2024 PHM Tutorial](2024PHMTutorial.ipynb)__. Unlike the previous models, the default parameters are for a Tattu battery. We can refer to the [documentation](https://nasa.github.io/progpy/api_ref/progpy/IncludedModels.html) for more details.\n",
```

Comment:
For this link to documentation, is there anything more specific we should point to?

Reply:
Meant the second link - will be in latest release